packages/UI/src/components/organisms/Navigation/MobileNavigation.tsx

```diff
@@ -58,7 +59,7 @@ const MobileNavigation = ({
                   <img
                     width="36"
                     height="33"
-                    src={user.profilePictureUrl}
+                    src={ProfilePhoto}
```

Comment:
This should be reverted as we're going to always show the logged-in user here rather than a static image